.ci/azure/windows.yml

```diff
@@ -25,6 +26,7 @@ pr:
     - '*.md'
     - '*/layer_tests_summary/*'
     - '*/conformance/*'
+    - 'tools/*'
 
```

Comment:
Here we have a test trying to install openvino-dev. I think we need to exclude MO at least here.
Can you do it please?

Reply:
I suppose we install openvino-dev here, but don't run any tests.

Reply:
Yes correct, but I think it is better to check that `openvino-dev` is installable on Windows. May be we make some requirements changes that affect it.
How do you think?

Reply:
I agree. Reverted.